scenarios1/14_Shadow_Empire.cfg

```diff
@@ -277,6 +275,10 @@
         [/message]
         [message]
             speaker=Efraim
+#ifdef HARD
+            message= _ "Well.  His arrogance is his greatest weakness.   It shall lead to his downfall on this field."
```

Comment:
The clue is not given on hard.

The extra space is probably there to average out all the other sentences preceded by only one space.

Reply:
This kind of surprise is not good for the gameplay. The scenario is much easier with the knowledge that Zorox should be fought first, and may not be even doable if one goes for the generals first, or fights the necromancers while they are supported by the generals.

It would really suck to fail the scenario a few times, resort to save scumming, and end up realising that it could be so easy if one went for Zorox first.

Reply:
Your call.  I think it's pretty clear Zorox is an objective, and since the scenario is so easy once you do so spelling it out for the player on hard is too much.  If there has to be a clue, I would like it at least to be a hint ("as quickly as possible" or something) than giving away the answer.  Or maybe the objectives could just list Zorox first.

I've never tried going after the generals first (I can't imagine why one would while excluding Zorox), but I have concentrated on the first necro and ignored Z in the beginning.  More work that way, but not bad (I'd consider this scenario harder than most if you don't target Z right away, but not by a lot).